test/Microsoft.ML.CpuMath.PerformanceTests/SsePerformanceTests.cs

```diff
@@ -106,5 +106,9 @@ public void MatMulX()
         [Benchmark]
         public void MatMulTranX()
             => SseIntrinsics.MatMulTran(src, src1, dst, 1000, 1000);
+
+        [Benchmark]
+        public void MatMulPX()
```

Comment:
Why is this test called `MatMulPX`, when the method it is benchmarking is named `MatMulP`?